tests/queries/0_stateless/02813_series_decompose.sql

```diff
@@ -0,0 +1,21 @@
+-- Tags: no-fasttest, no-cpu-aarch64
+-- Tag no-cpu-aarch64: values generated are slighly different on aarch64
+
+DROP TABLE IF EXISTS tb2;
+
+CREATE TABLE tb2 (`n` UInt32, `a` Array(Float64)) ENGINE = Memory;
+INSERT INTO tb2 VALUES (3,[10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34]);
+INSERT INTO tb2 VALUES (14, [139, 87, 110, 68, 54, 50, 51, 53, 133, 86, 141, 97, 156, 94, 149, 95, 140, 77, 61, 50, 54, 47, 133, 72, 152, 94, 148, 105, 162, 101, 160, 87, 63, 53, 55, 54, 151, 103, 189, 108, 183, 113, 175, 113, 178, 90, 71, 62, 62, 65, 165, 109, 181, 115, 182, 121, 178, 114, 170]);
+
+SELECT seriesDecomposeSTL([10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34, 10.1, 20.45, 40.34], 3);
+SELECT seriesDecomposeSTL([2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2], 0);
```

Comment:
Period = 0 sounds funny. How is that supposed to work?

Reply:
When a `signal`/time-series is flat( as in case of [2,2,2,2,2,2,2,2,2,2,2,2,2,2]), its period is 0. This is just an edge testcase.